src/components/Header.js

```diff
@@ -258,6 +260,21 @@ export default function _Header({ home }) {
       </OtherDropdownContainer>
     )
   }
+  
+  if(user && !user.emailVerified){
+    toast.error(
+      <div>
+        <p style={{marginBottom:10, fontWeight:'bold'}}>Welcome to Premiere!</p>
+        <p style={{marginBottom:10}}>Please verify your email address by clicking link sent to: <span style={{textDecoration: 'underline', color: 'blue' }}>{user.auth.currentUser.email}</span></p>
```

Comment:
This is a very long line and there are some styling inconsistencies, please make sure to run `prettier` before pushing